Applications/Spire/Source/BookView/BookQuoteTableModel.cpp

```diff
@@ -56,7 +56,7 @@ QVariant BookQuoteTableModel::data(const QModelIndex& index, int role) const {
     if(highlight.is_initialized()) {
       if(highlight->m_highlight_all_levels) {
         return highlight->m_color;
-      } else if(index.row() == (*m_market_first_index.find(market)).second) {
+      } else if(index.row() == (*m_first_market_index.find(market)).second) {
```

Comment:
Use ```->``` instead of ```*```

Reply:
Use ```[]```

Reply:
I think find was being used because the subscript operator is non-const.

Reply:
Okay that's good.